CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Change Log - AWS SDK for Android
 
+## [Release 2.18.0](https://github.com/aws/aws-sdk-android/releases/tag/release_v2.18.0)
+
+*Version 2.18.0 introduces breaking changes against 2.17.1.*
+
+### Breaking Updates
+- aws-android-sdk-mobile-client:
+   - No breaking changes but if you followed the old documentation to set up your code for HostedUI, you'll need to make the following revisions:
+   - Remove the intent filter from your Android Manifest talked about in [the old docs here](https://docs.amplify.aws/sdk/auth/hosted-ui/q/platform/android#setup-amazon-cognito-hosted-ui-in-android-app)
+   - Add the value for your redirect URI scheme ("myapp" in the docs example above) to your app build.gradle file as follows:
+   
+   ```gradle
+	android {
+	    defaultConfig {
+		manifestPlaceholders = [
+		    'authRedirectScheme': 'myapp'
+		]
+	    }
+	}
+   ```
+   - Remove the onResume intent callback code from your activity described in step 2 of the [the old docs here](https://docs.amplify.aws/sdk/auth/hosted-ui/q/platform/android#setup-amazon-cognito-hosted-ui-in-android-app)
+   - Add this onActivityResult override to handle the HostedUI callback instead:
+   
+   ```java
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+		super.onActivityResult(requestCode, resultCode, data);
+		if (requestCode == AuthClient.CUSTOM_TABS_ACTIVITY_CODE) {
+	    		AWSMobileClient.getInstance().handleAuthResponse(data);
+		}
+    }
+   ```
```

Comment:
indentation

Reply:
So it looks weird in the code but it's necessary for proper indentation in the render.

Reply:
```suggestion
```java
@Override
protected void onActivityResult(int requestCode, int resultCode, Intent data) {
    super.onActivityResult(requestCode, resultCode, data);
    if (requestCode == AuthClient.CUSTOM_TABS_ACTIVITY_CODE) {
        AWSMobileClient.getInstance().handleAuthResponse(data);
    }
}
```
```

Reply:
how does this look?

Reply:
Moved to docs update